src/scripting.h

```diff
@@ -23,9 +23,7 @@
 #include <string>
 #include <vector>
 
-#include <lua.h>
-#include <lauxlib.h>
-#include <lualib.h>
+#include <lua.hpp>
```

Comment:
I think this header does not solve the issue in #609, and we need some investigation to determine how to avoid memory leaks in c++ in luajit.

Reply:
http://luajit.org/extensions.html#exceptions

Reply:
Thanks, seems there is a detail comment: https://github.com/luajit/luajit/blob/master/src/lj_err.c#L20-L67

Reply:
> I think this header does not solve the issue in #609, and we need some investigation to determine how to avoid memory leaks in c++ in luajit.

@xiaobiaozhao you can try to call `DestroyState` in the Server destructor since it leaks by `cjson` open would init the buffer(1024). But I'm wondering why it happens in luajit only.

Reply:
The issus still exists. https://github.com/apache/incubator-kvrocks/pull/697/commits/707b177d60daf37dfcfbdc75a9a62ea4238a966d

Reply:
I can't reproduce in my ubuntu 20.4